docs/configure/themes.md

```diff
@@ -0,0 +1,115 @@
+* Once your images are in place you turn on custom images by changing the `System Artwork` setting to `Custom`
+
+#### For fullscreen artwork:
+
+* Create a folder in the path you created above called `artwork-fullscreen`
+* Upload your images to that folder
+* They can be named:
+    * `_default.png`
+    * `_default.jpg`
+    * `${system.theme}.png`
+    * `${system.theme}.jpg`
+    * The theme will look them them up in that order.
+    * `_default.png/jpg` can be used for creating a single image that is used for all systems OR a fallback for systems that you did not create a custom image for (if you don't want to use the fallback that already exists in the theme)
+    * `${system.theme}.png/jpg` should be named for the system you are looking to override. For example if you wanted to override the artwork for `snes` you would create an image called `snes.png` or `snes.jpg` in the artwork folder.
+* Once your images are in place you turn on custom images by changing the `System Artwork` setting to `Custom (Fullscreen)`
+
+### Color Schemes
+
+You can create your own custom color scheme to use for the theme
+
+* Download [this template](https://github.com/anthonycaccese/art-book-next-es/blob/main/resources/customizations/colors.xml)
+* Upload it in the path you created above and make sure its called `colors.xml`
+* Change any values in the template to the colors you prefer.
+* I tried to make the values as self explanatory as possible but if you have questions regarding which property does what please don't hesitate to ask.
+* After your colors are defined; in theme configuration change `Color Scheme` to `Custom`
+
+### Logos
+
+System logos can be customized by adding your own images
+
+* Create a folder in the path you created above called `logos`
+* Upload your images to that folder
+* They can be named:
+    * `${system.theme}.svg`
+    * `${system.theme}.png`
+    * The theme will look them them up in that order. If a given image is not found in your folder then the the images from the theme will be used as a fallback.  This allows you to customize only the images you want and still have images displayed for all systems.
+    * `${system.theme}.svg/png` should be named for the system you are looking to override. For example if you wanted to override the logo for `snes` you would create an image called `snes.svg` or `snes.png` in the logos folder.
+* Once your images are in place you turn on custom images by changing the `System Logos` setting to `Custom`    
+
+## Adding Additional Themes
+
+- Knulli's version of EmulationStation is sourced from the same version used in Batocera so you can also use themes that were created for Batocera directly
+- You can download themes directly using the built-in theme downloader (which includes nice previews of each theme)
```

Comment:
Maybe add instructions how to get to the theme downloader? Something like:

To find the theme downloader, press ++"Start"++ to bring up the main menu, go to the *Updates & Downloads* section, and choose *Themes*.